performance_tests/update_layers/update_xml_file.py

```diff
@@ -0,0 +1,55 @@
+"""
+Use this script
+
+COMMAND:
+python3 update_xml_file.py 'http://localhost:8080/geoserver/gwc/rest/layers/' USERNAME PASSWORD ./layers.txt ./tile_layer_template.xml.txt
+
+"""
+
+
+import sys
+import ast
+import json
+import requests
+import xmltodict
+from requests.auth import HTTPBasicAuth
+
+
+URL = sys.argv[1]
+username = sys.argv[2]
+password = sys.argv[3]
+layer_file = sys.argv[4]
+xml_file = sys.argv[5]
+
```

Comment:
please add checks to validate presence of correct arguments and, if something is not right, print the usage you put on top.